src/pages/SignUpMemberPage/SignUpMemberPage.jsx

```diff
@@ -22,58 +22,60 @@ const FormikTextField = ({ name, ...props }) => {
 
 const SignupMember = () => {
   const navigate = useNavigate();
-  const [error, setError] = useState('');
+  const [error, setError] = useState("");
   const { handleSignupWithEmailAndPassword, handleGoogleLogin } = useAuth();
 
   const handleLogin = async (type, values) => {
     try {
       switch (type) {
-        case 'google':
+        case "google":
           await handleGoogleLogin();
           break;
-        case 'email':
+        case "email":
           await handleSignupWithEmailAndPassword(values.email, values.password);
           break;
         default:
           break;
       }
-      navigate('/signup');
+      navigate("/signup");
     } catch (error) {
       setError(error.message);
       console.log(errorCode, errorMessage);
     }
   };
 
-  const initialValues = { email: '', password: '' };
+  const initialValues = { email: "", password: "" };
 
   const validationSchema = Yup.object().shape({
     email: Yup.string()
-      .email('Invalid email address')
-      .required('Email is required'),
-    password: Yup.string().required('Password is required'),
+      .email("Invalid email address")
+      .required("Email is required"),
+    password: Yup.string().required("Password is required"),
   });
 
   return (
     <Box
       sx={{
-        display: 'flex',
-        justifyContent: 'center',
-        alignItems: 'center',
-        height: '100vh',
-        padding: '20px',
+        display: "flex",
+        justifyContent: "center",
+        alignItems: "center",
+        height: "100vh",
+        padding: "20px",
       }}
     >
       <Stack
         alignItems="center"
         spacing={2}
-        sx={{ width: '100%', maxWidth: '360px', px: 2 }}
+        sx={{ width: "100%", maxWidth: "360px", px: 2 }}
       >
-        <Box
-          component="img"
-          src={Logo}
-          alt="Logo"
-          sx={{ width: '80%', maxWidth: '300px', height: 'auto', my: 4 }}
-        />
+        <a href="/homepage">
+          <Box
+            component="img"
+            src={Logo}
+            alt="Logo"
+            sx={{ width: "80%", maxWidth: "300px", height: "auto", my: 4 }}
+          />
+        </a>
```

Comment:
Since this is being used everywhere make the logo a component that routes to `/` and replace all the instances of it in the app

Reply:
changes made